portfolio/src/main/webapp/script.js

```diff
@@ -27,3 +27,25 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function getRandomQuote() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  document.getElementById('quote-container').innerText = quote;
+}
+
+function getComments() {
+  fetch('/data').then(response => response.json()).then((history) => {
```

Comment:
You may want to split the servlet into two and name them properly.
Creating single endpoint for all your tasks does not scale well.

Reply:
Do you mean creating another servlet.java to handle querying comments from DataStore?

Reply:
Yes. Of course it will have a different name too.